Automattic/wp-calypso
# Help Center: Hide placeholder message if waiting for AI response

Fixes: https://github.com/Automattic/dotcom-forge/issues/9776

## Proposed Changes

* Disables input field and changes placeholder text to `Just a moment...` when waiting for AI response

![image](https://github.com/user-attachments/assets/755b9a81-5028-4c63-948b-a6007360e6a8)


## Why are these changes being made?


* Hide input field placeholder while AI is responding.

## Testing Instructions



* Checkout branch or use live link
* Navigate to `/home?flags=help-center-experience` and open the Help Center
* Open a chat with AI and send a message.
* Verify the text input field placeholder changes to `Just a moment...` while the AI/Wapuu thinking icon is appearing (while waiting for AI response)
